exercises/G-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numOfStudents=15;
+let numOfMentors=8;
+let abs=numOfMentors+numOfStudents;
```

Comment:
How can you improve the _readability_ of your code here ? You've created a variable called `abs` but it's not immediately apparent to me what this means. Remember your code is not only read by a computer, but also by other people too! So remember to think carefully about your choice of variable names.